### PR TITLE
Adding a subtitle prop to the pageHeader component

### PR DESCRIPTION
**Pull request by alinevillaca**
Mobile
image

Desktop
image

### CODE REVIEW

File: react/components/PageHeader/index.js [hunk 1]
```diff
@@ -33,22 +33,34 @@ class PageHeader extends PureComponent {
...
 
-        <div
-          className={`vtex-pageHeader__title c-on-base t-heading-2 flex flex-row
-          flex-wrap justify-between ${linkLabel ? 'mt0' : 'mt7'}`}>
-          <span>{this.props.title}</span>
-
-          {children && (
-            <div className="vtex-pageHeader__children">{children}</div>
-          )}
+        <div>
```
Comment (klzns) [inline react/components/PageHeader/index.js]: This was added because of flex?

File: react/components/PageHeader/index.js [hunk 2]
```diff
@@ -33,22 +33,34 @@ class PageHeader extends PureComponent {
...
+            ${linkLabel ? 'mt0' : 'mt7'}`}>
+            <div className="t-heading-2 order-0 flex-grow-1">
+              {this.props.title}
+            </div>
+            {children && (
+              <div className="vtex-pageHeader__children order-2 order-0-ns mt5 mt0-ns">
+                {children}
+              </div>
+            )}
+            <div className="t-body lh-copy c-muted-1 mv5 order-1 order-0-ns">
```
Comment (klzns) [inline react/components/PageHeader/index.js]:
```suggestion
            <div className="vtex-pageHeader__subtitle t-body lh-copy c-muted-1 mv5 order-1 order-0-ns">
```

File: react/components/PageHeader/index.js [hunk 3]
```diff
@@ -33,22 +33,34 @@ class PageHeader extends PureComponent {
...
-          <span>{this.props.title}</span>
-
-          {children && (
-            <div className="vtex-pageHeader__children">{children}</div>
-          )}
+        <div>
+          <div
+            className={`vtex-pageHeader__title c-on-base flex flex-wrap flex-row justify-between
+            ${linkLabel ? 'mt0' : 'mt7'}`}>
+            <div className="t-heading-2 order-0 flex-grow-1">
```
Comment (klzns) [inline react/components/PageHeader/index.js]:
```suggestion
            <div className="vtex-pageHeader__title t-heading-2 order-0 flex-grow-1">
```

File: react/components/PageHeader/index.js [hunk 4]
```diff
@@ -33,22 +33,34 @@ class PageHeader extends PureComponent {
...
-          className={`vtex-pageHeader__title c-on-base t-heading-2 flex flex-row
-          flex-wrap justify-between ${linkLabel ? 'mt0' : 'mt7'}`}>
-          <span>{this.props.title}</span>
-
-          {children && (
-            <div className="vtex-pageHeader__children">{children}</div>
-          )}
+        <div>
+          <div
+            className={`vtex-pageHeader__title c-on-base flex flex-wrap flex-row justify-between
```
Comment (klzns) [inline react/components/PageHeader/index.js]:
```suggestion
            className={`c-on-base flex flex-wrap flex-row justify-between
```

File: react/components/PageHeader/index.js (hunk 1 above)
Comment (alinevillaca) [reply react/components/PageHeader/index.js]: It was, but became unnecessary. Removed.

**Pull request merged**